jabgui/src/main/java/org/jabref/gui/preferences/JabRefGuiPreferences.java

```diff
@@ -998,15 +999,32 @@ private NameDisplayPreferences.DisplayStyle getNameDisplayStyle() {
 
     // region: Main table, main table column, and search dialog column preferences
 
+    private MainTablePreferences getMainTablePreferencesFromLowLevelApi(MainTablePreferences defaults) {
+        return new MainTablePreferences(
+                getMainTableColumnPreferences(),
+                getBoolean(AUTO_RESIZE_MODE, defaults.getResizeColumnsToFit()),
+                getBoolean(EXTRA_FILE_COLUMNS, defaults.getExtraFileColumnsEnabled())
+        );
+    }
+
+    private ColumnPreferences getMainTableColumnPreferencesFromLowLevelApi(ColumnPreferences defaults) {
```

Comment:
Use `BackingStore` instead of `LowLevelApi`. We updated the issue description accordingly, but not yet the screenshot.